ci/code_checks.sh

```diff
@@ -229,6 +229,11 @@ if [[ -z "$CHECK" || "$CHECK" == "patterns" ]]; then
     invgrep -RI --exclude=\*.{svg,c,cpp,html,js} --exclude-dir=env "\s$" *
     RET=$(($RET + $?)) ; echo $MSG "DONE"
     unset INVGREP_APPEND
+
+    # Check if pandas is referenced as pandas, not *pandas* or Pandas
+    MSG='Checking if the pandas word reference is always used lowercase (pandas,not Pandas or PANDAS' ; echo  $MSG
+    invgrep -nr '*pandas*|Pandas' web/* doc/*
```

Comment:
I think the problem is with the astriks in the end, does the CI fail if we remove those?

Reply:
Let me check

Reply:
@MomIsBestFriend CI passes even after removing the asterisk